terraform/stages/foundation/main.tf

```diff
@@ -54,6 +67,16 @@ module "service_accounts" {
   project_number = data.google_project.project.number
 }
 
+module "shared_vpc" {
```

Comment:
where can we choose to use shared_vpc vs. regular vpc_network? Is it through the vars.shared_vpc?